### PR TITLE
add feature/navbar branch

### PR DESCRIPTION
**Pull request by jakanoh17**

### CODE REVIEW

File: src/components/BasicMenu/BasicMenu.jsx
Comment (markdreed512) [inline src/components/BasicMenu/BasicMenu.jsx:1]: The only feedback I have is the name of this component is a little confusing.  Perhaps DropdownMenu or DropdownNavItem?

File: src/components/App/App.css
Comment (markdreed512) [inline src/components/App/App.css:1]: Are all these formatting updates an autofix from linting? I suppose we should all get on the same page about that so we don't have this happen over and over.

File: src/components/BasicMenu/BasicMenu.jsx
Comment (jakanoh17) [reply src/components/BasicMenu/BasicMenu.jsx:1]: Changed the name to DropdownMenu. Thanks for the correction

**Comment by Valtaire-O** [comment]
Looks like theres some conflicts that need to be resolved here

**Comment by markdreed512** [comment]
Maybe you should pull the recent changes to the repo, resolve conflicts locally and then re-commit. Hopefully this will help with all the conflicts.

**Pull request merged**